core/resources/scene.yaml

```diff
@@ -110,8 +122,13 @@ layers:
         draw:
             polygons:
                 order: 1
-                color: '#fffffa'
+                #color: '#fffffa'
+                color: global.color.colorA
                 interactive: true # currently ignored
+            lines:
+                order: 2
+                color: function() { return global.color.colorE; }
```

Comment:
@blair1618 is right about how it should behave, however looking at the current JS flow, I realized this would not work, because the global was being replaced as a string before function compilation happens, so if `global.color.colorE` is a function, then `color: global.color.colorE` would work, but `color: function() { return global.color.colorE(); } would not (because `color: global.color.colorE` would still be a string). 

I'm updating JS behavior to enable this case, as I think it's the closest to what a typical users' expectations would be. It also enables some useful behavior, like color conversion functions, as global properties. These updates will be part of the JS 0.7 release.